src/cdrizzleapi.c

```diff
@@ -48,16 +48,160 @@ scale_image(PyArrayObject *image, double scale_factor) {
  * Top level function for drizzling, interfaces with python code
  */
 
+static int
+process_array_list(PyObject *list, integer_t *nx, integer_t *ny,
```

Comment:
Add comment to document what this function does, as well as what the function parameters are.